docs/src/main/sphinx/object-storage/metastores.md

```diff
@@ -182,6 +182,13 @@ properties:
 * - `hive.metastore.thrift.txn-lock-max-wait`
```

Comment:
please change the commit title to be more descriptive. Im ok with even squashing this to the first commit